store/mocks/DataStore.go

```diff
@@ -274,6 +277,34 @@ func (_m *DataStore) UnsetDeviceGroup(ctx context.Context, id model.DeviceID, gr
 	return r0
 }
 
+func (_m *DataStore) UnsetDevicesGroup(
```

Comment:
and where is the unit test for that? (nitpick mode on)

Reply:
ohh, there is a test in `store/mongo/datastore_mongo_test.go` but I called it `TestClearDevicesGroup` for some reason.